apps/src/music/views/MusicView.jsx

```diff
@@ -465,6 +465,14 @@ class UnconnectedMusicView extends React.Component {
       return;
     }
 
+    // Skip this pair of events to avoid extra compiles when dragging a block out of the toolbox.
+    if (
+      e.type === Blockly.Events.TOOLBOX_ITEM_SELECT ||
+      e.type === Blockly.Events.CREATE
+    ) {
+      return;
```

Comment:
Should we also cancel previews like we do with BLOCK_DRAG above?

Reply:
Good question.  It looks like we get a `BLOCK_DRAG` very shortly after, so I think we can rely on that.